test/execute_build_test.go

```diff
@@ -234,6 +285,18 @@ func testCreateImage(t *testing.T, _ spec.G, it spec.S) {
 		}, metav1.CreateOptions{})
 		require.NoError(t, err)
 
+		_, err = clients.client.KpackV1alpha2().Extensions(testNamespace).Create(ctx, &buildapi.Extension{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: extensionName,
+			},
+			Spec: buildapi.ExtensionSpec{
+				ImageSource: corev1alpha1.ImageSource{
+					Image: "natalieparellano/sample-extension", // FIXME
```

Comment:
Perhaps we could build the sample extension and push it along with all the other images from `hack/local.sh` or equivalent?

Reply:
I would prefer to use a properly released (read: maintained) extension, does CNB have a sample one? What about paketo or other well known buildpack authors?